src/desktopvirtualization/HISTORY.rst

```diff
@@ -6,3 +6,30 @@ Release History
 0.1.0
 ++++++
 * Initial release.
+
+0.2.0
+++++++
+* Bump version to 2021-07-12.
+* Update: host_pool_name max length up to 64 characters long to Fix [19298](https://github.com/Azure/azure-cli/issues/19298)
```

Comment:
>github [](http://example.com/codeflow?start=84&length=6)

Fix #19298: `az desktopvirtualization hostPool show`: Support the maximum length of the host pool name to 64 characters